ui/src/alerting/components/CheckHistoryVisualization.tsx

```diff
@@ -0,0 +1,78 @@
+// Libraries
+import React, {FC, createContext, useState} from 'react'
+import {get} from 'lodash'
+
+// Components
+import {Plot} from '@influxdata/giraffe'
+import CheckPlot from 'src/shared/components/CheckPlot'
+import EmptyQueryView, {ErrorFormat} from 'src/shared/components/EmptyQueryView'
+
+// Types
+import {Check, TimeZone, CheckViewProperties} from 'src/types'
+import TimeSeries from 'src/shared/components/TimeSeries'
+import {createView} from 'src/shared/utils/view'
+import {checkResultsLength} from 'src/shared/utils/vis'
+import {getTimeRangeVars} from 'src/variables/utils/getTimeRangeVars'
+import {TimeRange} from 'src/types'
+
+interface ResourceIDs {
```

Comment:
we are defining this interface in two places- would you mind defining it in one place and then importing it in the other?

Reply:
Im confused by this, are you referring to the time range, or the interface resource IDs?